Source/ScalingHeaderScrollView.swift

```diff
@@ -68,6 +80,9 @@ public struct ScalingHeaderScrollView<Header: View, Content: View>: View {
     /// Allow enlarging while pulling down
     private var allowsHeaderGrowthFlag: Bool = false
 
+    /// Use this variable to set the padding value of the content from the edge when pull-to-load is active
+    private var pullToLoadContentOffset: CGFloat = 0.0
```

Comment:
`pullToLoadMoreContentOffset`
you should always use the whole thing in every connected variable - `pullToLoadMore`: `pullToLoadMoreProgress` `shouldPullToLoadMore` etc. Don't throw away words to make it shorter, and don't rephrase to make it seemingly more fit to some exact use-case. Just use `pullToLoadMore` every time it has to do with this feature, then it will be much easier to maintain